test/mocha/dynamoDBLocalFixture.ts

```diff
@@ -0,0 +1,52 @@
+import AWS, { DynamoDB } from 'aws-sdk'
+import { ChildProcess } from 'child_process'
+import DDBLocal from 'dynamodb-local'
+import { deleteAllTables } from './dbSetup'
+
+process.env.AWS_ACCESS_KEY_ID = 'my_access_key'
+process.env.AWS_SECRET_ACCESS_KEY = 'my_secret_key'
+
+const dbPort = Number(process.env.DYNAMODB_LOCAL_PORT || 8000)
+
+let dbInstance: ChildProcess | undefined
+;(global as any)['__DYNAMODB_LOCAL__'] = true
+
+export const mochaGlobalSetup = async () => {
```

Comment:
I think we should wrap this all around a Try catch block to return a good error message saying that the DynamoDB Local was not instantiated (This is where things were failing when your Java version wasn't working, but it was not clear)

Reply:
ooh sg done